hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -404,6 +415,29 @@ forAllNonEmptyByronCommit action = do
           Right{} -> property False
           Left e -> action e
 
+forAllCommitWithMoreThan100ADA ::
+  (PostTxError Tx -> Property) ->
+  Property
+forAllCommitWithMoreThan100ADA action = do
+  forAll (genHydraContext maximumNumberOfParties) $ \hctx ->
+    forAll (genStInitial hctx) $ \(ctx, stInitial) ->
+      forAll genAdaOnlyUTxOOnMainnetWithAmountBiggerThan100k $ \utxo ->
+        case commit (alterChainContextNetworkToMainnet ctx) stInitial utxo of
+          Right{} -> property False
+          Left e -> action e
+ where
+  genAdaOnlyUTxOOnMainnetWithAmountBiggerThan100k :: Gen UTxO
+  genAdaOnlyUTxOOnMainnetWithAmountBiggerThan100k = do
+    vk <- arbitrary
+    adaAmount <- arbitrary `suchThat` (> 0)
+    let value = lovelaceToValue $ Lovelace (adaAmount + 100_000_000)
```

Comment:
why do we need to `+ 100_000_000` ?

is it not the same as restricting the adaAmount in the suchThat close?
```hs
adaAmount <- arbitrary `suchThat` (> 100_000_000)
```